.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "templates/bonsai-starter/lib/forge-std"]
```

Comment:
I think we generally should avoid submodule in the root project that submodule into the template. Should this .gitmodules be in the template directory?

Reply:
I'd like to avoid submodules entirely, can we discuss why this is necessary and possible alternatives?